web/template/course-card.gohtml

```diff
@@ -31,17 +42,16 @@
                             </template>
                         </div>
                     {{end}}
-                    <a class="text-5 font-sans text-sm ml-1" href="/w/{{.Slug}}/{{.GetNextLecture.Model.ID}}">
+                    <a class="text-5 font-sans text-sm ml-1" href="/w/{{$course.Slug}}/{{$course.GetNextLecture.Model.ID}}">
                         <i class="fas fa-angle-right"></i>
                         <span class="underline">Join waiting room</span>
                     </a>
                 {{else}}
-                    {{$next := .GetNextLecture.Start}}Next lecture: {{printf "%v %02d. %02d:%02d" $next.Month $next.Day $next.Hour $next.Minute}}
+                    {{$next := $course.GetNextLecture.Start}}Next lecture: {{printf "%v %02d. %02d:%02d" $next.Month $next.Day $next.Hour $next.Minute}}
                 {{end}}
             {{else}}
                 No upcoming Lectures
             {{end}}
```

Comment:
```suggestion
            {{end}}
            </p>
```

Reply:
Technically, this `</p>` should not be needed, but in general well-formed HTML is preferred.

Reply:
I think I deleted that without looking since GoLand suggested that the tag </p> matched nothing. I didn't see the starting tag on top.